t/16-utils.t

```diff
@@ -502,4 +503,17 @@ subtest 'human readable size' => sub {
     is(human_readable_size(245760), '240 KiB', 'less than a MB');
 };
 
+subtest 'create downloads list' => sub {
+    my $job_settings = {};
+    is_deeply create_downloads_list($job_settings), {}, 'empty for no settings provided';
+    $job_settings->{TEST_URL} = 'test';
+    $job_settings->{TEST} = 'test';
```

Comment:
How are these settings coming into play? They don't seem to make a difference on the surface.